manimlib/utils/tex_file_writing.py

```diff
@@ -67,20 +64,20 @@ def tex_to_dvi(tex_file):
         if exit_code != 0:
             log_file = tex_file.replace(".tex", ".log")
             raise Exception(
-                ("Latex error converting to dvi. " if not TEX_USE_CTEX
-                else "Xelatex error converting to xdv. ") +
+                ("LaTeX error converting to dvi. " if not use_ctex
+                else "XeLaTeX error converting to xdv. ") +
                 "See log output above or the log file: %s" % log_file)
     return result
 
 
-def dvi_to_svg(dvi_file, regen_if_exists=False):
+def dvi_to_svg(dvi_file, use_ctex=False, regen_if_exists=False):
```

Comment:
Should probably move `use_ctex` param to the right in order to not mess with positional arg order
```suggestion
def dvi_to_svg(dvi_file, regen_if_exists=False, use_ctex=False):
```

Reply:
Good point, thanks!